src/Data/Binary/FloatCast.hs

```diff
@@ -0,0 +1,54 @@
+
+{-# LANGUAGE FlexibleContexts #-}
+
+-- | This module is a literal copy of
+--   <http://hackage.haskell.org/package/reinterpret-cast-0.1.0/docs/src/Data-ReinterpretCast-Internal-ImplArray.html>.
+--
+--   Implements casting via a 1-elemnt STUArray, as described in
```

Comment:
You mean these? https://github.com/well-typed/binary-serialise-cbor/blob/master/Data/Binary/Serialise/CBOR/ByteOrder.hs#L221

Hopefully, `base` will export word to/from float methods some day...